ES/01_tour_hub_de_huggingface.md

```diff
@@ -0,0 +1,223 @@
+
+Alternativamente, puede solicitar que alguien del equipo de Hugging Face ejecute los tutoriales para su clase a través de la iniciativa [Gira de demo.cratización del ML](https://www.notion.so/ML-Demo-cratization-tour-with-66847a294abd4e9785e85663f5239652)!
+
+</aside>
+
+**Duración:** 20 a 40 minutos
+
+**Objetivo**: aprender a usar de manera eficiente el [Hub](http://huggingface.co) gratuito para poder colaborar en el ecosistema y dentro de equipos en proyectos de Machine Learning (ML).
+
+**Metas de aprendizaje:**
+
+- Conocer y explorar los más de 30,000 modelos compartidos en el Hub.
+- Aprender formas eficientes de encontrar modelos y datasets adecuados para su aplicación.
+- Aprender a trabajar de forma colaborativa.
+- Explorar demos de ML creados por la comunidad.
+
+**Formato:** Laboratorio corto o para llevar a casa
+
+**Público:** Estudiantes de cualquier origen interesados en usar modelos existentes o compartir los suyos.
+
+**Requisitos previos**
+
+- Comprensión de Machine Learning.
+- (Opcional, pero recomendado) Experiencia con Git ([recurso](https://learngitbranching.js.org/)).
+
+## ¿Por qué el Hub?
+
+El Hub es una plataforma central donde cualquiera puede compartir y explorar modelos, conjuntos de datos y demos de ML. El problema de "resolver la IA" no lo resolverá una sola empresa, sino una cultura de intercambio de conocimientos y recursos. Debido a esto, el Hub tiene como objetivo crear la colección más extensa de modelos, conjuntos de datos y demos de código abierto.
+
+Aquí hay algunos datos sobre el Hub de Hugging Face:
+
+- Hay más de 30,000 modelos públicos.
+- ¡Hay modelos para el procesamiento del lenguaje natural, la visión artificial, el audio/habla y el aprendizaje por refuerzo!
+- Hay modelos para más de 180 idiomas. El Español es uno de los lenguajes más presentes.
+- Cualquier biblioteca de ML puede aprovechar el Hub: desde TensorFlow y PyTorch hasta integraciones avanzadas con spaCy, SpeechBrain, Keras y otras 20 bibliotecas.
+
+### Explorando un modelo
+
+Empecemos la exploración de modelos. Puede acceder a 30,000 modelos (y creciendo) en [hf.co/models](http://hf.co/models). Verá a [gpt2](https://huggingface.co/gpt2) como uno de los modelos con más descargas. Hagamos clic en él.
+
+El sitio web lo llevará a la tarjeta del modelo cuando haga clic en un modelo. Una tarjeta modelo es una herramienta que documenta modelos, proporciona información útil sobre los modelos y es esencial para la detección y reproducibilidad.
+
+[https://www.youtube.com/embed/XvSGPZFEjDY](https://www.youtube.com/embed/XvSGPZFEjDY)
+
+La interfaz tiene muchos componentes así que vamos a repasarlos:
+
+- En la parte superior puede encontrar diferentes **etiquetas** para cosas como la tarea (_generación de texto, clasificación de imágenes, etc._), frameworks (_PyTorch, TensorFlow, etc._), el idioma del modelo (_inglés, árabe, etc._) y licencia (_por ejemplo, MIT_).
+
+![Untitled](../images/Untitled.png)
```

Comment:
I see my laziness with the Notion -> Mardown export is coming back to haunt me here :)

Since we might get future translations, what do you think about changing these filenames to something more descriptive? If that's OK with you, we'd also need to update the English source `.md` file

Reply:
Great! Applied.